### PR TITLE
Admiralty integration tutorial

### PR DESCRIPTION
**Pull request by ytsarev**
Provides short demo local setup of k8gb working
together with admiralty.io

### CODE REVIEW

**Review by donovanmuller — DISMISSED** [review]

**Review by kuritka — APPROVED** [review]

**Review by k0da — APPROVED** [review]
lgtm

File: docs/admiralty.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+kubectl --context kind-us get pod
+NAME                             READY   STATUS    RESTARTS   AGE
+podinfo-557c458ddb-9qpjf-l29w6   1/1     Running   0          11s
+
+kubectl --context kind-eu get pod
+NAME                             READY   STATUS    RESTARTS   AGE
+podinfo-557c458ddb-9nkmx-p7z9r   1/1     Running   0          15s
+```
+
+Observe that associated Ingress has also followed the Pods to the Target clusters
```
Comment (somaritane) [inline docs/admiralty.md:70]:
```suggestion
Observe that associated Ingress has also followed the Pods to the target clusters
```

File: docs/admiralty.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+podinfo   <none>   *                 80      10m
+
+kubectl --context kind-eu get ing
+NAME      CLASS    HOSTS   ADDRESS   PORTS   AGE
+podinfo   <none>   *                 80      10m
+```
+
+## Add k8gb annotations to Ingress object to enable global load balancing
+
+Observer that there are no Gslb resources in the Target clusters
```
Comment (somaritane) [inline docs/admiralty.md:84]:
```suggestion
Observer that there are no Gslb resources in the target clusters
```

File: docs/admiralty.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+
+```sh
+kubectl --context kind-eu get gslb
+No resources found in default namespace.
+
+kubectl --context kind-us get gslb
+No resources found in default namespace.
+```
+
+Add k8gb strategy annotation to the Ingress object in the Source cluster
```
Comment (somaritane) [inline docs/admiralty.md:94]:
```suggestion
Add k8gb strategy annotation to the Ingress object in the source cluster
```

File: docs/admiralty.md [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+
+## Create sample workloads on the source cluster
+
+```sh
+helm --kube-context kind-cd upgrade --install podinfo podinfo/podinfo --set replicaCount=2 --set ingress.enabled=true
+```
+
+## Add pod annotations to enable multi-cluster scheduling
+
+Edit `podinfo` Deployment to add admiralty `multicluster.admiralty.io/elect` scheduling Annotation to Pod spec template
```
Comment (somaritane) [inline docs/admiralty.md:38]:
```suggestion
Edit `podinfo` Deployment to add Admiralty `multicluster.admiralty.io/elect` scheduling Annotation to Pod spec template
```

**Review by somaritane — CHANGES_REQUESTED** [review]
lgtm, just several small fixes ahead.

File: docs/admiralty.md [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+## Deploy Admiralty
+
+Just follow https://admiralty.io/docs/quick_start
+
+## Deploy k8gb to target clusters
+
+```sh
+helm repo add k8gb https://www.k8gb.io
+
+kubectl --context kind-eu create ns k8gb
```
[the remark this answers is not in the document]
Comment (ytsarev) [reply docs/admiralty.md:19]: @somaritane it all based on https://admiralty.io/docs/quick_start/ as a prerquisite and kind is there

File: docs/admiralty.md [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+    metadata:
+      annotations:
+        prometheus.io/port: "9898"
+        prometheus.io/scrape: "true"
+        multicluster.admiralty.io/elect: ""
+      creationTimestamp: null
+...
+```
+
+Observe that pods were scheduled to Target cluster after then Annotation patch
```
[the remark this answers is not in the document]
Comment (ytsarev) [reply docs/admiralty.md:58]: or maybe vice versa for capital? As for
```
apiVersion: multicluster.admiralty.io/v1alpha1
kind: Target

```

File: docs/admiralty.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (somaritane) [reply docs/admiralty.md:19]: Ah, didn't notice!

File: docs/admiralty.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (somaritane) [reply docs/admiralty.md:58]: sure thing, just needs to be capital everywhere for consistency 👍

**Comment by ytsarev** [comment]
@somaritane thanks a ton for the thorough review, please check again

**Review by somaritane — APPROVED** [review]
lgtm!

**Review by kuritka — APPROVED** [review]

**Review by somaritane — APPROVED** [review]
lgtm

**Comment by ytsarev** [comment]
https://github.com/admiraltyio/admiralty/releases/tag/v0.13.2 ingress annotation trigger released so we can merge this doc

**Pull request merged**